### ISSUE TITLE
EmptyVolumeCache API ?

### ISSUE THREAD

**Issue by wingers999** [builtbybel/CleanmgrPlus#16]
Hi does this project use the same idea as burnbytes / comet e.g. the EmptyVolumeCache API ?

If so then can you explain how you can add additional locations to the standard "disk cleanup" locations, as you are doing with the use of script files?

I am very interested in learning more and contributing to the project but perhaps if it was open source then it would be a lot easier for us to see how it works and work towards creating a great community driven cleaner

**Comment by Belim** [comment]
No, here the EmptyVolumeCache API is not used and all cleaning options were mapped manually. 
Currently i also do not plan to release the source code of Cleanmgr+, sry!
